database/index.js

```diff
@@ -25,10 +25,12 @@ const getAuthor = id => search(`SELECT * FROM authors WHERE id =${id}`);
 
 const getAuthorTitles = id => search(`SELECT * FROM books WHERE author_id =${id}`);
 
+const updateStatus = (status, id) => search(`UPDATE books SET ? WHERE ?`, [{status: status}, {id: id}]);
+
```

Comment:
you can try using es6 template literals to pass values to the query string(try avoid putting space after equal signs *=${}* not *= ${}*) or you should pass the values in order"[value1, value2]" instead of passing an array of objects.

Reply:
I actually just got it working, so the current code is the working code. I'm not sure what I was doing wrong in my previous code(where I was using template literals) but I couldn't get it to work using template literals